cpp/src/pypowsybl.cpp

```diff
@@ -461,9 +461,9 @@ std::shared_ptr<load_flow_parameters> createLoadFlowParameters() {
 }
 
 LoadFlowComponentResultArray* runLoadFlow(const JavaHandle& network, bool dc, const std::shared_ptr<load_flow_parameters>& parameters,
-                                          const std::string& provider) {
+                                          const std::string& provider, const JavaHandle& reporter) {
```

Comment:
I was thinking: maybe instead of the NULL_HANDLE as we had discussed, we could simply use a nullable `JavaHandle*` here ?
I think this would allow the python binding to naturally translate `None` as a null `JavaHandle*`, removing some burden from the python side.

Reply:
You're right, thanks! Removed the python NULL_HANDLE, added a null-checking in the cpp part